docs/src/Formulation.md

```diff
@@ -356,24 +356,31 @@ where ``\Pi`` is known as the Exner function
 ```
 Note that the adiabatic exponent ``κ`` takes the effect of  moisture on the effective gas ``constant`` and specific heat capacity of air into account.
 
-### Virtual (Potential) Temperature
+### Virtual Temperature and Virtual Potential Temperature
 
-The virtual or density temperature ``T_v`` is the temperature dry air would need to have to have the same density as moist air at the same pressure. Using the ideal gas law ``p/ρ = R_m T``, this implies ``R_m T  = R_d T_v ``, or
+The virtual or density temperature ``T_v`` is the temperature dry air would need to have to have the same density as moist air at the same pressure.
+Using the ideal gas law ``p/ρ = R_m T``, this implies ``R_m T  = R_d T_v ``, or
 
 ```math
-\begin{equation}\label{e:virtual_temp}
+\begin{equation} \label{e:virtual_temp}
 T_v = \frac{R_m}{R_d} T.
 \end{equation}
 ```
 
 A virtual potential temperature can be defined analogously:
+
 ```math
 \begin{equation}
 θ_v = \frac{R_m}{R_d} θ.
 \label{e:virtual_pottemp}
 \end{equation}
 ```
-Some texts distinguish virtual and density (potential) temperatures, where density (potential) temperatures take the mass of condensate into account, whereas virtual (potential) temperatures do not. We always take the mass of any condensate into account in the thermodynamics of moist air and do not make this distinction here.
+
+!!! note
```

Comment:
```julia
!!! note "A title of the note"
```
?

Reply:
good idea

Reply:
except this change is really on #171, i accidentally combined the two